packages/terra-core-docs/src/terra-dev-site/doc/arrange/common/clinical-lowlight-theme/examplesetup.scss

```diff
@@ -0,0 +1,11 @@
+// stylelint-disable scss/at-mixin-pattern
+@import '~terra-mixins/lib/Mixins';
+
+.clinical-lowlight-theme {
```

Comment:
why do we need lowlight-theme file here ..??

Reply:
The interactive elements which i added like home, menu and settings button etc are in black colour. These buttons are not visible when i switched to clinical theme. For that i have added clinical theme scss file and added svg icons with white color.